lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -776,6 +777,15 @@ enum {
 using namespace llvm;
 
 namespace SPIRV {
+typedef SPIRVMap<dwarf::SourceLanguage, spv::SourceLanguage> SPIRVDWARFSourceLanguageMap;
+template <>
+inline void SPIRVDWARFSourceLanguageMap::init() {
```

Comment:
This is a bidirectional map and it can't have duplicated values.

Reply:
I've changed the mapping to use two switch statements instead.